app-creator/src/widgets/tool-bar/tool-bar.ts

```diff
@@ -4,6 +4,12 @@
  */
 import { LitElement, html, customElement, css } from 'lit-element';
 import '@polymer/paper-button/paper-button.js';
+import { store } from '../../redux/store';
+
+// The paper dialog widget appears on export button click and is used to display the serialized template string.
```

Comment:
Sorry I wasn't being clear. I was wondering if you could add a comment about what the `ToolBar` component is used for? This comment for paper dialog could be moved to `openDialog()` I think.